### DISCUSSION TITLE
Vs Code 扩展插件 生成 追加 返回参数

### DISCUSSION THREAD

**Discussion by ByteXiong** [alovajs/alova#459] [Feature Request]
### What problem does the new feature need to solve?

Vs Code 扩展插件  在handleApi 配置中responses 配置 追加参数
比如API 返回 一个**List集合**里 存在 **相对路径的图片** 
前端需要 把循环 集合 给每张图片加上 **http 绝对路径**
就需要 在**集合对象内追加** 一个字段存 **绝对路径**

### How important is it to you?

Better to have

**Comment by JOU-amjs** [comment]
这个不是扩展插件处理的事，插件所使用的openapi文件描述的是接口数据结构以及描述的集合，具体的返回数据是在实际访问接口时获取的，需要在项目代码中处理，以下有两个建议：

1. 全局处理：如果你有很多接口都需要处理成绝对路径，并且有一定规则，比如固定放在imageList字段中，那你可以在`alova`的全局拦截器里处理。
2. 如果只是某几个接口需要处理，那可以在对应接口的`transformData`（v3为`transform`）中手动处理。
